pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -83,6 +86,66 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+type TabPanelProps = {
+  children: any
+  value: any
+  index: any
+}
+
+function TabPanel(props: TabPanelProps) {
+  return (
+      <Typography
+          component="div"
+          role="tabpanel"
+          hidden={props.value !== props.index}
+          id={`scrollable-auto-tabpanel-${props.index}`}
+          aria-labelledby={`scrollable-auto-tab-${props.index}`}
+      >
+        <Box p={3}>{props.children}</Box>
+      </Typography>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired
+};
```

Comment:
nits, we may not need this, if only `children` is unrequired; adding a `?` after the field name in the type declaration is enough.

Reply:
For sure, as long as we use Typescript, PropTypes doesn't come into play basically (except for [this](https://github.com/Microsoft/TypeScript/issues/4833). Let me revise it.